backend/app.ts

```diff
@@ -1,40 +1,35 @@
 import express, { NextFunction, Request, Response } from "express";
 import path from "path";
+const multer = require("multer");
+const upload = multer();
 
-import { companyRouter } from "./routes/companyRouter";
+import { employeeRouter } from "./routes/employeeRouters";
 import { DBAuthenticationError } from "./error/500s";
 import { statusCodedError } from "./error/statusCodedError";
-const cors = require('cors')
-
+const cors = require("cors");
 
 // -------------------firing express app
 const app = express();
+
+//body paramter enable
+app.use(upload.array());
+
 app.use(express.json());
 app.use(cors());
 app.use(express.urlencoded({ extended: false }));
 app.use(express.static(path.join(__dirname, "client/build")));
 
 // -------------------routes
-app.use("/company", companyRouter);
+// app.use("/company", companyRouter);
 app.get("/home", (request: Request, response: Response) => {
-  console.log(request.url);
+  console.log(request.query);
   response.json({ message: `Welcome to the home page!!` });
 });
+app.use("/employee", employeeRouter);
 
+// ------------ error handling. It only has 500 error, but later more errors will be handled.
 app.use(function (err: Error, req: Request, res: Response, next: NextFunction) {
-  if (err instanceof statusCodedError) {
-    res.status(err.getStatusCode()).send(err.name);
-  } else {
-    res.status(500).send(err.name);
-  }
-
-  if (err instanceof DBAuthenticationError) {
-    console.error(
-      "DB Authentication Error: please check DATABASE_URL in .env file"
-    );
-    console.error(err.message);
-  }
-
+  res.status(500).send(err.stack);
```

Comment:
JSON error?

Reply:
500 means a general server error. This means that users do everything they are supposed to do, but our backend or database sucks.

Reply:
I mean, I send for more convenient development, but yeah it is a bad idea to send the whole error stack lol.

Reply:
Actually, I think I should change it back to console.error